src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -716,34 +802,77 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper
         return true;
     }
 
+
+    bool Config() {
```

Comment:
I think you should replace all of this with the new way of getting the plug-in config. Look at XrdOucGatherConf.hh which pretty much replaces all the manual handling of the config file. I'd prefer to start using this for all new code.

Reply:
Good idea.  Done in the latest commits.